<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>384</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>388</width>
    <height>249</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MFIT - Configurações de Detecção</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QListWidget" name="listDetectConfig" >
     <property name="minimumSize" >
      <size>
       <width>81</width>
       <height>231</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>81</width>
       <height>231</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <item>
      <property name="text" >
       <string>Corte</string>
      </property>
      <property name="icon" >
       <iconset>
        <normaloff>../Interface/Icons/cut.png</normaloff>../Interface/Icons/cut.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Fade</string>
      </property>
      <property name="icon" >
       <iconset>
        <normaloff>../Interface/Icons/fades.png</normaloff>../Interface/Icons/fades.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Dissolve</string>
      </property>
      <property name="icon" >
       <iconset>
        <normaloff>../Interface/Icons/dissolve.png</normaloff>../Interface/Icons/dissolve.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="cutPage" >
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QWidget" native="1" name="cutWidget" >
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QGroupBox" name="groupBox" >
            <property name="minimumSize" >
             <size>
              <width>243</width>
              <height>54</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>243</width>
              <height>54</height>
             </size>
            </property>
            <property name="title" >
             <string>Limiar</string>
            </property>
            <layout class="QGridLayout" >
             <item row="0" column="2" >
              <widget class="QSpinBox" name="spinPercentage" >
               <property name="maximum" >
                <number>100</number>
               </property>
               <property name="value" >
                <number>45</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLabel" name="labelPercentage" >
               <property name="text" >
                <string>Porcentagem:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QGroupBox" name="groupBox_2" >
            <property name="minimumSize" >
             <size>
              <width>243</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>243</width>
              <height>80</height>
             </size>
            </property>
            <property name="title" >
             <string>Filtro de Canny</string>
            </property>
            <layout class="QGridLayout" >
             <item row="1" column="1" >
              <widget class="QSpinBox" name="spinLastCanny" >
               <property name="maximum" >
                <number>255</number>
               </property>
               <property name="value" >
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QSpinBox" name="spinFirstCanny" >
               <property name="maximum" >
                <number>255</number>
               </property>
               <property name="value" >
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="labelLast" >
               <property name="text" >
                <string>Final:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="labelFirst" >
               <property name="text" >
                <string>Inicial:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dissolvePage" />
     <widget class="QWidget" name="fadePage" />
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="helpButton" >
     <property name="text" >
      <string>Ajuda</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
